screens/TabOneScreen.tsx

```diff
@@ -1,13 +1,36 @@
-import * as React from 'react';
+import React, { useState } from 'react';
```

Comment:
The last time I tried this, this camera wasn't really working, so we can revert the changes to this file.